testing/features/support/pages/error_summary.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module DesignSystem
+  class ErrorSummary < SitePrism::Page
+    set_url "/iframe.html?id=3-components--error-summary&viewMode=story"
+
+    section :initial_form, "#a11yComponentToTest" do
+      element :heading, "#error-summary-title"
+      element :error_quantity_label, ".cads-error-summary__field_count"
```

Comment:
Is there any way in these kinds of tests to query based on the display text? Using CSS classes isn't ideal

Reply:
We use https://testing-library.com/ in the client-side tests for this reason, is there an equivalent in cucumber land?

Reply:
I'd prefer not to query based on text. Given we often assert based on text. Although having said that, this is a sample testing library, so perhaps we could?

If there are ID's to be hooked into they're perfect. I cannot remember if there was. I'll do a quick pass to check